Organic/AdsTxt.php

```diff
@@ -21,25 +22,26 @@ public function __construct( Organic $organic ) {
         add_action( 'init', array( $this, 'show' ) );
     }
 
-    public function get() {
-         return $this->organic->getOption( 'organic::ads_txt' );
+    public function getAdsTxtUrl() {
```

Comment:
Looks like there is one more place where `ads.txt` was used
https://github.com/orgnc/wordpress-plugin/blob/feature%2FEP-822%2Fplugin-ads-redirect/Organic/GraphQL.php#L144

This GQL API is used by `organic-web`
https://github.com/orgnc/organic-web/blob/master/src/lib/wordpress/api/fragments/organicConfigFragment.ts

Looks like this field is actually not used there, but you should check with Content Team to be sure and ask them about how they handle the `ads.txt`